src/internalHelpers/_hslToRgb.js

```diff
@@ -22,9 +22,9 @@ function hslToRgb(
   }
 
   // formular from https://en.wikipedia.org/wiki/HSL_and_HSV
-  const huePrime = hue % 360 / 60
+  const huePrime = (((hue % 360) + 360) % 360) / 60 // Corrected for potential negative hue
```

Comment:
Instead of an inline comment, might be worth putting a note in the docs on how we handle the potential negative hue.

Reply:
I fixed the typo and removed the comment. I'll try looking into the docs later, when I have a bit more time.